### PR TITLE
Turn estimated_duration into an object with unit and value

### PR DESCRIPTION
**Pull request by psrpinto**
Following this discussion: https://github.com/teamleadercrm/api/pull/276#pullrequestreview-204258740

Once we agree on this, I will proceed to using it in the other documentation PRs we have open.

Note that the main `apiary.apib` file is not updated here because the Tasks section isn't being rendered, since we don't want to release it yet.

### CODE REVIEW

**Comment by rathesDot** [comment]
This looks a bit strange. Is the build missing something?

**Comment by psrpinto** [comment]
@rathesDot I just added this to PR description: 

> Note that the main `apiary.apib` file is not updated here because the Tasks section isn't being rendered, since we don't want to release it yet.

**Review by WouterSioen — APPROVED** [review]

**Review by rathesDot — APPROVED** [review]

**Comment by rathesDot** [comment]
Won't that lead to any problems when others try to build their things?

**Comment by psrpinto** [comment]
@rathesDot Should have no impact, the `tasks.apib` file is just not being "imported" anywhere yet. Once we're ready to release, we'll import it in `src/apiary.apib`. 

We did this so that we could iterate on the API docs without actually releasing the Tasks endpoints. Once the docs are out there, the endpoints are officially released and we can no longer break backwards compatibility.

We plan on releasing these endpoints by the end of this sprint.

**Comment by rathesDot** [comment]
Thanks for explaining @regularjack!

File: apiary.apib [hunk 1]
```diff
@@ -314,7 +314,7 @@ This documentation only reflects the latest version of the API.
 If you need to make changes to your client or you want to make use of the latest feature, you might need to upgrade your API version.
 
 You can find your current API version, in the `X-Api-Version` header on any API response.
-The current latest version is **2018-10-30**.
+The current latest version is **2019-01-24**.
```
Comment (stivni) [inline apiary.apib:317]: 👏 well spotted

File: src/08-tasks/tasks.apib [hunk 2]
```diff
@@ -61,7 +61,11 @@ Create a new task.
         + description (string, required)
         + due_at: `2016-02-04T16:44:33+00:00` (string, required)
         + work_type_id: `32665afd-1818-0ed3-9e18-a603a3a21b95` (string, required)
-        + estimated_duration: 3600 (number, optional) - In seconds
+        + estimated_duration (object, optional)
```
Comment (stivni) [inline src/08-tasks/tasks.apib:64]: 👍 I like this a lot

**Review by stivni — COMMENTED** [review]
~don't forget to build this~ scratch that

File: apiary.apib (hunk 1 above)
Comment (WouterSioen) [reply apiary.apib:317]: That's just the `build` that should've run on master 😬

**Review by stivni — APPROVED** [review]

File: Makefile [hunk 3]
```diff
@@ -11,7 +11,7 @@ lint: node_modules
 build: src node_modules
 	docker run -it --init --rm -v $(PWD):/code -w /code node:9 npm run build
 
-preview: node_modules
+preview: build
```
Comment (tiagobastos) [inline Makefile:14]: YES!! Then you can just do `make previews && open apiary.html` 😋

**Review by tiagobastos — APPROVED** [review]

**Review by dietrichm — APPROVED** [review]

**Comment by regularjack** [comment]
DO NOT MERGE, waiting for implementation (I should have made it a draft PR).

**Review by r1dotl — APPROVED** [review]

**Comment by psrpinto** [comment]
Merging since we need this for other PRs, even though the code hasn't yet been adapted to this format, but that's fine since this API isn't yet released.

**Pull request merged**